framework/include/actions/Action.h

```diff
@@ -280,17 +152,14 @@ class Action : public ConsoleStreamInterface,
 
   /// Timers
   PerfID _act_timer;
-};
-
-template <typename T>
-const T &
-Action::getParam(const std::string & name) const
-{
-  return InputParameters::getParamHelper(name, _pars, static_cast<T *>(0));
-}
 
-bool
-Action::isParamSetByUser(const std::string & name) const
-{
-  return _pars.isParamSetByUser(name);
-}
+  using MooseBase::_app;
+  using MooseBase::_name;
+  using MooseBase::_type;
+  using MooseBaseErrorInterface::_console;
+  using MooseBaseErrorInterface::mooseDeprecated;
+  using MooseBaseErrorInterface::mooseError;
+  using MooseBaseErrorInterface::mooseWarning;
+  using MooseBaseParameterInterface::_action_factory;
+  using MooseBaseParameterInterface::_factory;
```

Comment:
why these? Are you changing the access level?

Reply:
It seems I am because I needed this. Well this explains that, I just need to get the access level right in the base classes instead